web/modules/api/src/api.ts

```diff
@@ -104,27 +100,57 @@ class WikijumpAPIInstance extends Api<void> {
   }
 
   /**
-   * Returns a base URL but for a different subdomain.
+   * Helper for sending a `GET` request via the API.
    *
-   * @param subdomain - The subdomain to use.
+   * @param to - The path to send the request to.
+   * @param query - The query parameters to send, if any.
    */
-  subdomainURL(subdomain: string) {
-    return `${window.location.protocol}//${subdomain}.${window.location.host}/${API_PATH}`
+  get<T = void>(to: string, query: Record<string, string> = {}) {
```

Comment:
Since this is the return type, shouldn't it be:
```suggestion
  get<T = void>(to: string, query: Record<string, string> = {}): T {
```
?

Reply:
TypeScript infers automatically.